paddle/fluid/operators/detail/sendrecvop_utils.cc

```diff
@@ -39,7 +39,9 @@ void SerializeToByteBuffer(const std::string& name, framework::Variable* var,
   // parallelism execution, need to know when to free the tensor.
   DestroyCallback destroy_callback = [](void* backing) {};
 
-  void* buf = malloc(1024);
+  auto buffer = std::unique_ptr<char[]>(new char[1024]);
```

Comment:
I think it's ok for now. Is it possible to statically create it? Maybe it's faster?
char char_buf[1024];
void* buffer = reinterpret_cast<void*>(char_buf)?

Reply:
Maybe it change to be larger later? ^_^